lib/inject.js

```diff
@@ -37,16 +37,19 @@ function findFramesAndInject(parent, script, driver) {
  * @private
  * @param  {WebDriver}   driver   Instance of WebDriver to inject into
  * @param  {String}	  axeSource   Optional Attest source and configure string
+ * @param  {Object}   config      Optional configure object to pass to iframes
  * @param  {Function} callback    Callback to execute when aXe has been injected
  */
-module.exports = function(driver, axeSource, callback) {
+module.exports = function(driver, axeSource, config, callback) {
 	axeSource = axeSource || require('axe-core').source;
 
+	var branding = {branding:{application:"webdriverjs"}};
+	var configSrc = JSON.stringify(Object.assign(branding, config || {}));
 	var script = '(function () {' +
 		'if (typeof axe === "object" && axe.version) { return; }' +
 		'var s = document.createElement("script");' +
```

Comment:
You could kill two birds with one stone by getting rid of the `<scriipt>` tag creation here